subprojects/testing-jvm/src/main/java/org/gradle/api/internal/tasks/testing/junit/JUnitSpec.java

```diff
@@ -22,11 +22,13 @@
     private final Set<String> includeCategories;
     private final Set<String> excludeCategories;
     private final Set<String> includedTests;
+    private final String requestClass;
 
-    public JUnitSpec(Set<String> includeCategories, Set<String> excludeCategories, Set<String> includedTests) {
+    public JUnitSpec(Set<String> includeCategories, Set<String> excludeCategories, Set<String> includedTests, String requestClass) {
```

Comment:
Well, I didn't know what you guys prefer. I'm little bit worried about further extension (more String parameters) of this class in case of two constructors as they can quickly become ambigous.

Do you really want me to change that? I have another change prepared :)